app/helpers/ebird_helper.rb

```diff
@@ -0,0 +1,125 @@
+require 'faraday'
+require 'json'
+include ActionView::Helpers::NumberHelper
+
+module EbirdHelper
+  def getBirdData(lat,lng, num_req=100, num_ret=1, rare=false)
+    ebird_params = {  :lat => number_with_precision(lat, precision: 2),
+                      :lng => number_with_precision(lng, precision: 2),
+                      :maxResults => num_req,
+                      :back => 10,
+                      :dist => 25,
+                    }
+
+    url = "https://api.ebird.org/v2/data/obs/geo/recent/"
+    if rare == true
+      url = "https://api.ebird.org/v2/data/obs/geo/recent/notable"
+    end
```

Comment:
can you ternary operator here:
url = rare ? "https://api.ebird.org/v2/data/obs/geo/recent/notable" : "https://api.ebird.org/v2/data/obs/geo/recent/" 
or standard if else case I think would be more readable